bundles/org.openhab.binding.argoclima/README.md

```diff
@@ -0,0 +1,423 @@
+| schedule3OnTime        | text            | The time of day (HH:MM) the device should turn **ON** *(in the last used mode)* on the ```schedule3DayOfWeek```-specified days. In effect only if ```activeTimer``` [channel](#channels) is in ```SCHEDULE_TIMER_3``` mode                                     | 11:00                                | no       | yes      |
+| schedule3OffTime       | text            | The time of day (HH:MM) the device should turn **OFF** on the ```schedule3DayOfWeek```-specified days. In effect only if ```activeTimer``` [channel](#channels) is in ```SCHEDULE_TIMER_3``` mode                                                              | 22:00                                | no       | yes      |
+| resetToFactoryDefaults | boolean(action) | When set, upon successful Thing initialization, the binding will issue a one-time factory reset request to the device (and flip this value back do OFF)                                                                                                        | false                                | no       | yes      |
+
+## Channels
+
+Both thing types are functionally equivalent and support the same channels.
+
+| Channel                | Type                 | Read/Write | Description                                                                                                                                                                                                            |
+|------------------------|----------------------|------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| - **A/C Controls**                                                                                                                                                                                                                                                               ||||
+| power                  | Switch               | RW         | This is the control channel                                                                                                                                                                                            |
+| mode                   | String               | RW         | Operation mode. One of: ```COOL```, ```DRY```, ```FAN```, ```AUTO```                                                                                                                                                   |
+| setTemperature         | Number:Temperature   | RW         | The device's target temperature                                                                                                                                                                                        |
```

Comment:
All channel Id's should be `lower-case-hyphen`
https://www.openhab.org/docs/developer/guidelines.html#naming-convention